package.json

```diff
@@ -3,16 +3,16 @@
   "version": "0.0.1",
   "description": "Project Platform Health",
   "scripts": {
-    "build": "neutrino build",
-    "start": "neutrino start",
-    "start:local": "BACKEND=http://localhost:3000 neutrino start",
-    "lint": "neutrino lint",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open",
+    "start:local": "BACKEND=http://localhost:3000 webpack-dev-server --mode development --open",
+    "test": "jest",
+    "lint": "eslint --cache --ext mjs,jsx,js --format codeframe src",
```

Comment:
btw ff you wanted to lint the JS dotfiles in the repo root, you could change this to:

`... --format codeframe \".*.js\" src`